src/Chapter2.hs

```diff
@@ -620,7 +640,11 @@ Write a function that takes elements of a list only on even positions.
 >>> takeEven [2, 1, 3, 5, 4]
 [2,3,4]
 -}
-takeEven = error "takeEven: Not implemented!"
+takeEven :: [a] -> [a]
+takeEven [] = []
+takeEven [x] = [x]
+takeEven (x : _ : xs) = [x] ++ takeEven xs
```

Comment:
Same suggestion as for `duplicate`: `x : someList` is usually better then `[x] ++ someList`